freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -68,6 +72,16 @@ def handle(self, event):
 
         msg = "Skipping due to being blocked on further implementation for now."
```

Comment:
I would move this to right above `db_event.transition` line since it's not used until then. It makes the code easier to read since there is less context the reader needs to retain.